rules/scripting-user-agents.data

```diff
@@ -1,9 +1,21 @@
 # Generic HTTP clients (popular libraries)
 
+# http library
+# https://docs.aiohttp.org/en/stable/
+# User-Agent: Python/VERSION aiohttp/VERSION
+aiohttp/
 # http library
 # http://search.cpan.org/~opera/HTTP-DAV/DAV.pm
 dav.pm/v
 # http library
+# https://pkg.go.dev/net/http
+# User-Agent: Go http package
+# User-Agent: Go 1.1 package http
+# User-Agent: Go-http-client/VERSION
+Go http package
+Go 1.1 package http
+Go-http-client/
```

Comment:
Hmmm... 🤔 what happenss with Prometheus and similar software that is doing monitoring?

Reply:
> Hmmm... 🤔 what happenss with Prometheus and similar software that is doing monitoring?

I think the WAF should be used to protect the external interface, not the internal interface.

Reply:
Why should `Go` get better treatmen than `libwww-perl` and python?

Reply:
Just because this is PL1 and we try to avoid FPs as much as possible.

Reply:
I don’t think it should get a special treatment. It’s very easy to change the User-Agent header after 2 hours into Go (example: https://github.com/lifeforms/httpcheck/blob/master/httpcheck/init.go#L35) so any serious client should change it...